src/osgWrappers/serializers/osg/ProxyNode.cpp

```diff
@@ -109,16 +109,20 @@ struct ProxyNodeFinishedObjectReadCallback : public osgDB::FinishedObjectReadCal
     {
         osg::ProxyNode& proxyNode = static_cast<osg::ProxyNode&>(obj);
 
+        if (proxyNode.getDatabasePath().empty())
+        {
+            osgDB::FilePathList& fpl = ((osgDB::ReaderWriter::Options*)is.getOptions())->getDatabasePathList();
```

Comment:
I's inappropriate to just case away constness with a C cast.  A InputStream is const for a reason - loaders can be run multi-threaded.  osgDB::Options supports a cloneOptions() method that should be used if you wish to use the Options elswhere such as assigning it to another object.